### PR TITLE
CMS-1624 Move FAS homepage and contact in international

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: great_international/apps.py [hunk 1]
```diff
@@ -26,6 +26,8 @@ def ready(self):
         cache.InvestHighPotentialOpportunityDetailPageSubscriber.subscribe()
         cache.InvestHighPotentialOpportunityFormPageSubscriber.subscribe()
         cache.InvestHighPotentialOpportunityFormSuccessPageSubscriber.subscribe()  # noqa
+        cache.InternationalTradeHomePageSubscriber.subscribe()
```
Comment (MichaelWalker) [inline great_international/apps.py]: I don't think the word 'International' adds anything here. Is there anything to stop us just having `Trade...`?

File: makefile [hunk 2]
```diff
@@ -144,8 +144,6 @@ update_db_template: \
 	debug_migrate
 	pg_dump \
 		--no-owner \
-		--exclude-table=auth_user \
-		--exclude-table=users_userprofile \
 		--file=db_template.sql \
```
Comment (richtier) [inline makefile]: what about excluding the table data?

File: makefile [hunk 3]
```diff
@@ -144,8 +144,6 @@ update_db_template: \
 	debug_migrate
 	pg_dump \
 		--no-owner \
-		--exclude-table=auth_user \
-		--exclude-table=users_userprofile \
```
Comment (MichaelWalker) [inline makefile]: Was deleting these lines deliberate? They were added quite recently and seemed like a sensible addition!

File: makefile (hunk 2 above)
Comment (sdonk) [reply makefile]: ops

File: makefile (hunk 3 above)
Comment (sdonk) [reply makefile]: They produce an unusable template. `--exclude-table-data` is the right flag to use

**Pull request merged**